centreon/www/install/php/Update-23.10.0.php

```diff
@@ -356,7 +356,8 @@
                         ('centreon-widget-generictext', '23.10.0'),
                         ('centreon-widget-singlemetric', '23.10.0'),
                         ('centreon-widget-graph', '23.10.0'),
-                        ('centreon-widget-topbottom', '23.10.0')
+                        ('centreon-widget-topbottom', '23.10.0'),
+                        ('centreon-widget-statusgrid', '23.10.0')
```

Comment:
This won't work if a platform already is in 23.10.0
-> for develop and dev-23.10.x: Update-23.10.2.php or Update-23.10.3.php (depending of release status for 23.10)
-> for develop:  Update-24.10.0.php

Reply:
I don't know, the update file is not created